Python/Product/Uap/Interpreter/PythonUapInterpreter.cs

```diff
@@ -0,0 +1,237 @@
+﻿/* ****************************************************************************
+ *
+ * Copyright (c) Microsoft Corporation. 
+ *
+ * This source code is subject to terms and conditions of the Apache License, Version 2.0. A 
+ * copy of the license can be found in the License.html file at the root of this distribution. If 
+ * you cannot locate the Apache License, Version 2.0, please send an email to 
+ * vspython@microsoft.com. By using this source code in any fashion, you are agreeing to be bound 
+ * by the terms of the Apache License, Version 2.0.
+ *
+ * You must not remove this notice, or any other, from this software.
+ *
+ * ***************************************************************************/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PythonTools.Analysis;
+using Microsoft.PythonTools.Interpreter;
+using Microsoft.VisualStudioTools;
+
+namespace Microsoft.PythonTools.Uap.Interpreter {
+    class PythonUapInterpreter : IPythonInterpreter, IPythonInterpreterWithProjectReferences2, IDisposable {
```

Comment:
Adding references doesn't actually do anything other than provide IntelliSense, unless you've got something in your .targets to pull them in (typically they need to be part of a Python environment anyway, so there's little value and much confusion in doing this). Better to prevent adding references at all.

We don't currently, but we ought to hide the References node if the project doesn't support it. Right now you've got a change to omit the node it looks like, but we can generalise that better.